mu4/scores/internal/openscorecontroller.cpp

```diff
@@ -23,19 +23,29 @@
 
 using namespace mu;
 using namespace mu::scores;
+using namespace mu::domain::notation;
 
 void OpenScoreController::init()
 {
     dispatcher()->reg(this, "file-open", this, &OpenScoreController::openScore);
     dispatcher()->reg(this, "file-import", this, &OpenScoreController::importScore);
+    dispatcher()->reg(this, "file-newscore", this, &OpenScoreController::newScore);
 }
 
-void OpenScoreController::openScore()
+void OpenScoreController::openScore(const actions::ActionData &path)
```

Comment:
Something is wrong with this argument. Its name is "path", however its type is ActionData